plugins/qtosgrave/osgviewerwidget.cpp

```diff
@@ -1355,10 +1353,10 @@ void QOSGViewerWidget::Zoom(float factor)
     if (IsInOrthoMode()) {
         // if we increase _currentOrthoFrustumSize, we zoom out since a bigger frustum maps object to smaller part of screen
         _currentOrthoFrustumSize = _currentOrthoFrustumSize / factor;
-        _SetCameraViewOrthoProjectionPlaneSize(_currentOrthoFrustumSize);
+        _SetCameraViewOrthoProjectionPlaneSize(ClampDistance(_currentOrthoFrustumSize));
```

Comment:
frustum should have no effects on camera distance. when zoomig in ortho mode, the camera distance does not change